packages/tsdl/tsdl.0.8.2/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "Daniel Bünzli <daniel.buenzl i@erratique.ch>"
+authors: ["Daniel Bünzli <daniel.buenzl i@erratique.ch>"]
+homepage: "http://erratique.ch/software/tsdl"
+doc: "http://erratique.ch/software/tsdl/doc/Tsdl"
+dev-repo: "http://erratique.ch/repos/tsdl.git"
+bug-reports: "https://github.com/dbuenzli/tsdl/issues"
+tags: [ "audio" "bindings" "graphics" "media" "opengl" "input" "hci" ]
+license: "BSD-3-Clause"
+available: [ ocaml-version >= "4.00.1" ]
+depends: [ "ocamlfind" "ctypes" {>= "0.4.1"} "ctypes-foreign" ]
```

Comment:
Uuuuuh yes sorry it seems the notification slipped in my emails. Constraint added to the opam file of the project. Very important for this package since it has outrageous build times, apparently it hits some kind of slow path in the compiler (unclear what it is exactly).